staging/src/k8s.io/apiserver/pkg/util/flowcontrol/metrics/metrics.go

```diff
@@ -295,13 +295,11 @@ var (
 	)
 	apiserverWorkEstimatedSeats = compbasemetrics.NewHistogramVec(
 		&compbasemetrics.HistogramOpts{
-			Namespace: namespace,
-			Subsystem: subsystem,
-			Name:      "work_estimated_seats",
-			Help:      "Number of estimated seats (maximum of initial and final seats) associated with requests in API Priority and Fairness",
-			// the upper bound comes from the maximum number of seats a request
-			// can occupy which is currently set at 10.
-			Buckets:        []float64{1, 2, 4, 10},
+			Namespace:      namespace,
+			Subsystem:      subsystem,
+			Name:           "work_estimated_seats_ratio",
+			Help:           "Ratio of number of estimated seats (maximum of initial and final seats) associated with requests and its maximum value in API Priority and Fairness",
+			Buckets:        []float64{0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 0.95, 0.99, 1},
```

Comment:
I am not sure the bounds 0.95 and 0.99 are particularly valuable here.  We have them for utilization metrics where the denominator is something allocated by the administrator and it is interesting to know how often the allocation gets used up.  That is not quite the situation here.

Reply:
True, removed.